images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -177,8 +177,8 @@ function write_deployment_info() {
 
 # Prepare appliance initialization environment
 function prepare_init_env() {
-  # Make a copy of CONTAINER_DATA_PERSIST_FILE into PV if not present
-  [ ! -f "${PV_DATA_PERSIST_FILE}" ] && cp -a "${CONTAINER_DATA_PERSIST_FILE}" "${APP_ROOT_PERSISTENT}"
+  # Copy CONTAINER_DATA_PERSIST_FILE into PV ensuring it is kept up to date
+  cp -a "${CONTAINER_DATA_PERSIST_FILE}" "${APP_ROOT_PERSISTENT}"
```

Comment:
Is there a reason we have to have this be user modifiable? @fbladilo ?

I'm worried that if we want to add something to this file, that will never get used because we only copy over to the PV if the file doesn't already exist. So existing installations will only ever persist what was in the file on first deploy.

If we need the customer to be able to persist additional things I would rather have a separate file just for that and always read this one from the image so that we will always have the most up to date list of files to persist.